src/processor/graph-update-completion-monitor.service.ts

```diff
@@ -40,8 +41,9 @@ export class GraphUpdateCompletionMonitorService extends BlockchainScannerServic
     private readonly schedulerRegistry: SchedulerRegistry,
     blockchainService: BlockchainService,
     private readonly cacheService: ReconnectionCacheMgrService,
+    eventEmitter: EventEmitter2,
   ) {
-    super(cacheManager, blockchainService, new Logger(GraphUpdateCompletionMonitorService.name));
+    super(cacheManager, blockchainService, new Logger(GraphUpdateCompletionMonitorService.name), eventEmitter);
```

Comment:
Failing here

Reply:
I forgot to reset that file. Those changes weren't needed in the end.

Reply:
Removed file